src/components/@shared/AssetTeaser/AssetTeaser.tsx

```diff
@@ -51,7 +54,11 @@ export default function AssetTeaser({
           </div>
 
           <footer className={styles.foot}>
-            <Price accessDetails={asset.accessDetails} size="small" />
+            <Price
+              accessDetails={asset.accessDetails}
+              loading={loading}
```

Comment:
If we take this approach here, then we may as well do the same on the asset detail page so that the "No price set" message doesn't pop up there

Reply:
mmmm.... what do you mean? Do you see any "No price set" on the detail page? You shouldn't 

https://user-images.githubusercontent.com/13741335/166895382-c7064545-8dfd-4814-800e-c8ddb072be7c.mp4

Reply:
Yeah, I'm seeing it show "No price set" for a second before the price loads

https://user-images.githubusercontent.com/20739535/166912417-1c868cdb-185e-4da9-8ace-d2367ec2478a.mp4

Reply:
> Yeah, I'm seeing it show "No price set" for a second before the price loads
> 
>  Berlin.Fish.Fauna.2013.-.Ocean.Market.-.5.May.2022.mp4

Interesting. I'll take a look at it as you shouldn't see it.

Reply:
Ok cool, thanks

Reply:
pushed fix now